### DISCUSSION TITLE
[Feature request] Free domains, DDNS and URL shorteners in blocked services

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#2732] [Q&A]
Hello,

please, add three new types to blocking services list:
1. Free domains (.TK, .ML, .GA, .CF, .GQ, .FOR.UZ, .PP.UA, .CO.CC etc.)
2. Dynamic DNS (.HOPTO.ORG, .ZAPTO.ORG, .SYTES.NET, .DDNS.NET, NO-IP.COM etc.);
3. URL shorteners (BIT.LY, GOO.GL, CLCK.RU, ADF.LY etc.).

Maybe someone will advise the most complete lists of these domains🤨

Thank you.
Cheers🤙

**Comment by ameshkov** [comment]
It does not fit the "services" list. But if there're already any blocklists for this kind of domains, we could add them to #1913

**Comment by Aikatsui** [comment]
@ammnt that also possible to go to here https://github.com/AdguardTeam/AdGuardHome/issues/2230 I noticed it has "parked domains" sub-category.
(I say as it because suggested that categories list, after reviewed several third-party services)

**Comment by DandelionSprout** [comment]
To block free domains: https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Alternate%20versions%20Anti-Malware%20List/AntiMalwareAdGuardHome.txt

To block dynamic DNS: https://raw.githubusercontent.com/stamparm/maltrail/master/trails/static/suspicious/dynamic_domain.txt

To block URL shorteners: https://raw.githubusercontent.com/cbuijs/shallalist/master/urlshortener/domains
